public/500.html

```diff
@@ -59,8 +59,8 @@ <h2>There was an error. <small>Are you sure you have the right address?</small><
             <li>your PublicLab.org username (if you have one)</li>
           </ul>
 
-          <p><a class="btn btn-primary" href="https://github.com/publiclab/plots2/issues/new">Report a bug on Github</a> or send an email to <a href="mailto:web@publiclab.org">web@publiclab.org</a> with the above information.</p>
-
+            <p><a class="btn btn-primary" href="https://github.com/publiclab/plots2/issues/new" target="_blank">Report a bug on Github</a> <a class="btn btn-primary" href="https://oss.skylight.io/app/applications/GZDPChmcfm1Q/recent/6h/endpoints" target="_blank">Check if the site is running slow</a> or, send an email to <a href="mailto:web@publiclab.org">web@publiclab.org</a> with the above information.</p>
```

Comment:
@scheleon , The changes look good !! 
Can u correct the indentation of this line ? Thanks :)